lib/metrics/context.js

```diff
@@ -96,6 +101,15 @@ module.exports = function (log, config) {
           data.flowBeginTime = metadata.flowBeginTime
           data.flowCompleteSignal = metadata.flowCompleteSignal
           data.flowType = metadata.flowType
+
+          const doNotTrack = this.headers && this.headers.dnt === '1'
```

Comment:
Should the `DNT` apply to flow metrics related tracking as well?

Reply:
Not to my knowledge.